TravelBug/src/Components/VideoCard/VideoCard.jsx

```diff
@@ -1,16 +1,22 @@
 import "./VideoCard.css";
-
+import { useNavigate } from "react-router-dom";
 const VideoCard = ({ vid }) => {
+  const navigate = useNavigate();
   const { title, category, img, creator, _id } = vid;
+  const toSingleVideoPage = () => {
+    navigate(`/singlevideo/${_id}`);
+  };
   return (
     <>
       <div className="video-card">
-        <div className="image-container">
+        <div onClick={() => toSingleVideoPage()} className="image-container">
```

Comment:
Since the function does not have any parameters to be passed, this can just be onClick={toSingleVideoPage}

Reply:
Done